### PR TITLE
docs: add hyperlinks to directories and improve navigation

### PR DESCRIPTION
**Pull request by young010101**
**End-user friendly description of the problem this fixes or functionality that this introduces**

- [ ] Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

---
**Link of any specific issues this addresses**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -57,16 +57,16 @@ If you work for a company that provides a cloud-based runtime, you could help us
 by implementing the [interface specified here](https://github.com/All-Hands-AI/OpenHands/blob/main/openhands/runtime/base.py).
 
 #### Testing
-When you write code, it is also good to write tests. Please navigate to the `tests` folder to see existing test suites.
-At the moment, we have two kinds of tests: `unit` and `integration`. Please refer to the README for each test suite. These tests also run on GitHub's continuous integration to ensure quality of the project.
+When you write code, it is also good to write tests. Please navigate to the [`./tests`](./tests) folder to see existing test suites.
+At the moment, we have two kinds of tests: [`unit`](./tests/unit) and `integration`. Please refer to the README for each test suite. These tests also run on GitHub's continuous integration to ensure quality of the project.
```
[the remark this answers is not in the document]
Comment (young010101) [reply CONTRIBUTING.md:61]: Thanks for the suggestion! Would `./tests/runtime` be the right link for `integration`? If not, could you point me in the right direction? Still learning the file structure by going through the docs and code—great project, by the way!

**Comment by young010101** [comment]
> Thank you for the contribution! A few comments.

Thanks for the docs philosophy insight!

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (young010101) [reply CONTRIBUTING.md:61]: > It'd be good to add "integration" too
@openhands-agent Would ./tests/runtime be the right link for integration? If not, could you point me in the right direction? IMPORTANT: The only thing you need to do is adding the proper hyper link for `integration`.

**Comment by young010101** [comment]
@openhands-agent 
Would ./tests/runtime be the right link for integration? If not, could you point me in the right direction? IMPORTANT: The only thing you need to do is adding the proper hyperlink for `integration` in [CONTRIBUTING.md](./CONTRIBUTING.md).

**Comment by enyst** [comment]
@young010101 Sorry, if you want to see how fun openhands-agent is in real use, please take a look at this guide to set it up:
[OpenHands Github Issue Resolver](https://github.com/All-Hands-AI/OpenHands/tree/main/openhands/resolver#using-the-github-actions-workflow)

It will work on your repositories, or repositories where you have permissions to push. For example, you can set it up and call it - like you did - on your repo https://github.com/young010101/OpenHands

**Comment by enyst** [comment]
(Lets try with the label!)

Integration tests have been moved out of the tests directory in openhands repo. Find them and fix the CONTRIBUTING.md documentation as pointed out in comments above.

**Comment by enyst** [comment]
It failed. 🤔 

I posted an issue on it: https://github.com/All-Hands-AI/OpenHands/issues/5292

I suspect it failed because your branch is external to the repository (a fork) - for now.

Back to your question: integration tests really were in the same place, near unit tests, but they have been rewritten and moved, and now they are here:
https://github.com/All-Hands-AI/OpenHands/tree/main/evaluation/integration_tests

**Comment by young010101** [comment]
> @young010101 Sorry, if you want to see how fun openhands-agent is in real use, please take a look at this guide to set it up: [OpenHands Github Issue Resolver](https://github.com/All-Hands-AI/OpenHands/tree/main/openhands/resolver#using-the-github-actions-workflow)
> 
> It will work on your repositories, or repositories where you have permissions to push. For example, you can set it up and call it - like you did - on your repo https://github.com/young010101/OpenHands

Thank you for the clarification and the link to the guide. I'll review the setup steps and see how it works on my repository. Appreciate the guidance!

**Comment by young010101** [comment]
> It failed, lol. 🤔
> 
> I posted an issue on it: #5292
> 
> I suspect it failed because your branch is external to the repository (a fork) - for now, that doesn't seem to work.
> 
> Back to your question: integration tests were in the same place, indeed a part of /tests, but they have been rewritten and moved, and now they are here: https://github.com/All-Hands-AI/OpenHands/tree/main/evaluation/integration_tests

Thanks for identifying the issue and sharing the new location of the integration tests! I’ll adjust my branch and workflow accordingly.

**Review by enyst — APPROVED** [review]
Thank you!

**Pull request merged**